all_protos/injective/evm/v1/state.proto

```
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package injective.evm.v1;
```

Buf (1.54.0)

Files with package "injective.evm.v1" must be within a directory "injective/evm/v1" relative to root but were in directory "all_protos/injective/evm/v1".

(PACKAGE_DIRECTORY_MATCH)